redwoods-insurance-salesforce/force-app/main/default/lwc/incidentImageCarousel/__tests__/incidentImageCarousel.test.js

```diff
@@ -17,6 +30,47 @@ describe('c-incident-image-carousel', () => {
         document.body.appendChild(element);
 
         const labelElement = element.shadowRoot.querySelector('p');
-        expect(labelElement.textContent).toContain('no images');
+        expect(labelElement.textContent).toContain(
+            'There are currently no images of damage for this case.'
+        );
+    });
+
+    it('renders two pictures when the wire mock returns two', () => {
+        // Create initial element
+        const element = createElement('c-incident-image-carousel', {
+            is: IncidentImageCarousel
+        });
+        document.body.appendChild(element);
+
+        // Emit data from @wire
+        getRecordAdapter.emit('mockRecordId');
+        getRelatedPicturesAdapter.emit(mockTwoImages);
```

Comment:
It should be tested that `recordId` as well as `IMAGE` for `fileType` are set on the wire call (using `getLastConfig`).

Reply:
done.